src/BuoyancyModels/buoyancy.jl

```diff
@@ -1,21 +1,21 @@
-using Oceananigans.Grids: ZDirection, validate_unit_vector
+using Oceananigans.Grids: NegativeZDirection, validate_unit_vector
 
 struct Buoyancy{M, G}
     model :: M
     gravity_unit_vector :: G
 end
 
 """
-    Buoyancy(; model, gravity_unit_vector=ZDirection())
+    Buoyancy(; model, gravity_unit_vector=NegativeZDirection())
 
-Uses a given buoyancy `model` to create buoyancy in a model. The optional keyword argument 
-`gravity_unit_vector` can be used to specify the direction opposite to the gravitational
-acceleration (which we take here to mean the "vertical" direction).
+Uses a given buoyancy `model` to create buoyancy in a model. The optional keyword argument
+`gravity_unit_vector` can be used to specify the direction of gravity (default `NegativeZDirection()`).
+The buoyancy acceleration acts in the direction opposite to gravity.
 
 Example
 =======
 
-```jldoctest
+```jldoctest; filter = r".*tartarus.*"
```

Comment:
Note that for some reason this filtering passes.

Reply:
Just note that if somebody runs the doctests on their machine they'll fail because probably it won't be called tartarus :)

Reply:
Hmm, is there another solution... ?

Reply:
Good point @navidcy. For the record I don't know it'll fail because I think the filtering is based off of what's in the `doctest`, not on the output (and `tartarus` is in the foctest). But you're right that it's best to be safe.

I'll change both instances to something safer once we figure out how to make documentation pass.

Reply:
Yes, ` ```jldoctest buoyancy; filter = [r".*┌ Warning.*", r".*└ @ Oceananigans.*"] `

Reply:
[3dfa7d2](https://github.com/CliMA/Oceananigans.jl/pull/2963/commits/3dfa7d2881df166a08f31a680dcdc37413b91f21)

Reply:
hopefully it works -- I'll fix it if not